feature-group-definitions/device-orientation-events.dist.yml

```diff
@@ -0,0 +1,31 @@
+# Generated from: device-orientation-events.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: Device orientation events (intial support)
+# TODO: Add a note that coordinates can differ noticeably between platforms and devices
+description: The `DeviceMotion` and `DeviceOrientation` events report the movement and orientation of the browser's device in physical space.
+spec: https://w3c.github.io/deviceorientation/
+caniuse: deviceorientation
+status:
+  baseline: low
+  baseline_low_date: 2023-09-18
+  support:
+    chrome: "31"
+    chrome_android: "31"
+    edge: "12"
+    firefox: "6"
+    firefox_android: "6"
+    safari: "17"
```

Comment:
What was the final piece here? Your old markdown tables would be very useful to look at during review :D

Reply:
All of it: `DeviceMotionEvent`, `DeviceOrientationEvent`, and the window events themselves.

And a lot of that scripting used the precursor to compute-baseline—I'll see if I can quickly reinstate that.

Reply:
As an alternative, how about just adding a comment after each BCD key with its Baseline date in a comment?